utils/demo/helpers/createImageIdsAndCacheMetaData.js

```diff
@@ -51,7 +52,12 @@ export default async function createImageIdsAndCacheMetaData({
   }
 
   const modality = instances[0][MODALITY].Value[0];
-  let imageIds = instances.map((instanceMetaData) => {
+
+  const sortedInstances = Object.values(instances).sort(
+    (a, b) => a[INSTANCE_NUMBER].Value[0] - b[INSTANCE_NUMBER].Value[0]
+  );
```

Comment:
I think you will be interested in checking out this helper: `utilities.sortImageIdsAndGetSpacing`

Reply:
I already tested it and it's not right.

![image](https://github.com/cornerstonejs/cornerstone3D/assets/50026015/6434abe3-62e1-4fd6-8ef4-f1566e14c5bc)

```javascript
async function fetchDicom() {
    // Get Cornerstone imageIds for the source data and fetch metadata into RAM
    imageIds = await createImageIdsAndCacheMetaData(dev.getConfig.fetchDicom);

    //
    const { sortedImageIds } = csUtilities.sortImageIdsAndGetSpacing(imageIds);

    //
    await loadDicom(sortedImageIds.reverse());
}
```